pages/api/feedContent/[id].ts

```diff
@@ -0,0 +1,144 @@
+import type {
+  FeedComment,
+  FeedItemContent,
+} from "../../../components/feed/FeedPage";
+
+const api_url =
```

Comment:
```suggestion
const API_URL =
```

to signal it's a constant